client/x/mint/types/params.go

```diff
@@ -26,7 +27,8 @@ func DefaultParams() Params {
 	return Params{
 		MintDenom:         sdk.DefaultBondDenom,
 		InflationsPerYear: math.LegacyNewDec(24625000000000000.000000000000000000),
-		BlocksPerYear:     uint64(60 * 60 * 8766 / 5), // assuming 5 second block times
+		BlocksPerYear:     uint64(60 * 60 * 8766 / 5), // assuming 5 seconds block times
+		SingularityHeight: 725760,                     // 42 days with assuming 5 seconds block time
```

Comment:
Should we combine this with the other SingularityHeight in evmstaking to make it consistent?

Reply:
If possible, it would be better to combine into one, but I have no idea on combining this. Or one way for this is that we make a new keeper named `singularity` and import the keeper to both `mint` and `evmstaking` keeper. I think there is no need to make a new keeper for this, thus make like this. Do you have any better idea on this?